### PR TITLE
Implement IM-08-10 and refactor documents and views

### PR DESCRIPTION
**Pull request by tilokowalski**

### CODE REVIEW

File: src/main/java/com/fes/hazelnut/schoolnut/view/page/allocation/TeacherPage.java [hunk 1]
```diff
@@ -16,11 +22,14 @@
 import com.vaadin.flow.server.auth.AnonymousAllowed;
 import com.vaadin.flow.theme.lumo.LumoUtility;
 
+import lombok.extern.slf4j.Slf4j;
```
Comment (tilokowalski) [inline src/main/java/com/fes/hazelnut/schoolnut/view/page/allocation/TeacherPage.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'lombok.extern.slf4j.Slf4j'

File: src/main/java/com/fes/hazelnut/schoolnut/view/page/AllocationPage.java [hunk 2]
```diff
@@ -19,10 +20,15 @@
...
 import com.vaadin.flow.component.orderedlayout.VerticalLayout;
 import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.router.BeforeEnterEvent;
+import com.vaadin.flow.router.BeforeEnterObserver;
 import com.vaadin.flow.router.PageTitle;
 import com.vaadin.flow.router.Route;
 import com.vaadin.flow.server.auth.AnonymousAllowed;
 import com.vaadin.flow.theme.lumo.LumoUtility;
+
+import java.time.LocalDate;
```
Comment (tilokowalski) [inline src/main/java/com/fes/hazelnut/schoolnut/view/page/AllocationPage.java:30]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.time.LocalDate'

**Review by VogtJoshua — APPROVED** [review]

File: src/main/java/com/fes/hazelnut/schoolnut/view/page/AllocationPage.java [hunk 3]
```diff
@@ -33,32 +37,73 @@
...
+        setSizeFull();
+
+        add(new H1("Kehren Sie später zurück"));
+
+        add(new Paragraph(
+            dateEnrollment == null
+            ? "Aktuell ist kein Einschulungstermin festgelegt. Bitte versuchen Sie es später erneut."
+            : "Der Einschulungstermin ist noch nicht erreicht. Die Klassenzuordnung findet unter Aufsicht der Lehrkräfte vor Ort statt, bitte versuchen Sie es dann erneut."
+        ));
+
```
Comment (thisami) [inline src/main/java/com/fes/hazelnut/schoolnut/view/page/AllocationPage.java:100]: Sehr sexy gemacht!
Da ist mir fast einer abgegangen 😆

**Review by thisami — APPROVED** [review]
Sehr geil gemacht

**Review by lenav99 — APPROVED** [review]

File: src/main/java/com/fes/hazelnut/schoolnut/view/page/AllocationPage.java (hunk 3 above)
Comment (tilokowalski) [reply src/main/java/com/fes/hazelnut/schoolnut/view/page/AllocationPage.java:100]: kuss maus

File: src/main/java/com/fes/hazelnut/schoolnut/document/pdf/EnrollmentFormDocument.java [hunk 4]
```diff
@@ -16,19 +20,9 @@
 public class EnrollmentFormDocument extends PdfDocument {
 
     /**
-     * The font used for enrollment headings.
+     * The config service to for performing database operations on the config.
```
[the remark this answers is not in the document]
Comment (tilokowalski) [reply src/main/java/com/fes/hazelnut/schoolnut/document/pdf/EnrollmentFormDocument.java:23]: egal

**Pull request merged**